gpytorch/math/functions/lazy_kronecker_product/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .kp_interpolated_toeplitz_gp_marginal_log_likelihood import KPInterpolatedToeplitzGPMarginalLogLikelihood
+from .kp_toeplitz_mvn_kl_divergence import KPToeplitzMVNKLDivergence
+
+__all__ = [
+    KPInterpolatedToeplitzGPMarginalLogLikelihood,
+    KPToeplitzMVNKLDivergence
+]
```

Comment:
Also, these files now belong in `gpytorch/functions`, not `gpytorch/math/functions`.